pocs/utils/logger.py

```diff
@@ -45,25 +46,34 @@ def get_root_logger(profile='panoptes', log_config=None):
     # Get log info from config
     log_config = log_config if log_config else load_config('log').get('logger', {})
 
+    invoked_script = os.path.basename(sys.argv[0])
     log_dir = '{}/logs'.format(os.getenv('PANDIR', '/var/panoptes/'))
-    log_fname = 'panoptes-{}-{}.log'.format(os.getpid(), datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
-    log_fname_generic = 'panoptes.log'
+    log_fname = '{}-{}-{}'.format(invoked_script, os.getpid(),
+                                  datetime.datetime.utcnow().strftime('%Y%m%dT%H%M%SZ'))
+    log_fname_generic = '{}'.format(invoked_script)
 
     # Alter the log_config to use UTC times
     if log_config.get('use_utc', True):
         for name, formatter in log_config['formatters'].items():
             log_config['formatters'][name].setdefault('()', _UTCFormatter)
 
-    log_file_lookup = {
-        'all': "{}/{}".format(log_dir, log_fname),
-        'warn': "{}/warnings.log".format(log_dir),
-    }
-
-    # Setup the TimeedRotatingFileHandler to backup in middle of day intead of middle of night
+    # Set log filename and rotation
     for handler in log_config.get('handlers', []):
-        log_config['handlers'][handler].setdefault('filename', log_file_lookup[handler])
-        if handler in ['all', 'warn']:
-            log_config['handlers'][handler].setdefault('atTime', datetime.time(hour=11, minute=30))
+        # Set the filename
```

Comment:
Is it necessary with the Python logger system to create all of the files up front, along with those symlinks, or can we defer until the first message is written to the handler?

Reply:
Note that I'm only creating the symlink for the `all` file (which I suppose should just be called `debug`). But I'm actually not sure about the actual question. We immediately start writing some messages so the `all` file is created right away.  I'll check to see order.